modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -49,19 +48,21 @@ If you have external client providers such as PingFederate configured for your e
 
 *Best Practice*
 
-To secure your APIs by leveraging the multiple client providers feature, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. 
+To secure your APIs, create one external client provider per environment. Assign the existing client provider to your production environment and add the new providers to other environments, such as sandbox, and QA. However, check the possible impact on managed APIs and their consumers. 
 
-However, before changing any existing client providers, ensure that you check the impact on managed APIs and their consumers. Before you implement multiple client providers, see the <<guidelines,guidelines>>.
+Avoid using the same IdP in production and non-production environments. You can use the same IdP in multiple production environments, or in multiple non-production environments.
 
+Before implementing multiple client providers, see the <<guidelines,guidelines>>.
```

Comment:
Again, this seems to be an afterthought when it sounds like more of a prerequisite.

[I am remembering now that I have commented about these things with the previous writer of this text; hoping that we can make some structural changes along the way....]

Reply:
Thanks, Kim. Good point! I'm thinking that the Guidelines section doesn't really list prerequisites, but rather it lists best practices.